# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../ifplugd/linux-2.6.18.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/interface.c.orig	2006-10-24 16:02:09.000000000 +0300
+++ ./src/interface.c	2006-10-24 16:02:33.000000000 +0300
@@ -22,6 +22,7 @@
 #include <config.h>
 #endif
 
+#include <stdint.h>
 #include <linux/sockios.h>
 #include <linux/if_ether.h>
 #include <sys/types.h>
